### PR TITLE
Added 'Upgrade Building' text

### LINKED ISSUE #23: [Feature Request] Add text before upgrade buttons to clarify their purpose
image

### PR DESCRIPTION
**Pull request by HanzCEO**
|Before|After|
|--|--|
</img> | </img>

Fixes #23

### CODE REVIEW

File: src/scripts/ui/BuildingUpgradeComponent.tsx [hunk 1]
```diff
@@ -59,22 +59,29 @@ export function BuildingUpgradeComponent({ gameState, xy }: IBuildingComponentPr
...
-                     ).join(", ")}`}
-                     placement="top"
-                  >
-                     <button key={level} className="f1" onClick={() => upgrade(level)}>
-                        x{level}
-                     </button>
-                  </Tippy>
-               ))}
+            <div className="column">
+               <div className="mb5 text-strong text-small">Upgrade Building</div>
```
Comment (jimmyh77) [inline src/scripts/ui/BuildingUpgradeComponent.tsx:63]: You might be better off adding the title to the separator.  And you also need to add the text to the translation file (en.ts)

```
               <div className="separator has-title">
                  <div>
                     {t(L.UpgradeBuilding)}
                  </div>
               </div>
```

Comment (HanzCEO) [reply src/scripts/ui/BuildingUpgradeComponent.tsx]: | English | Russian |
| -- | -- |
| image | image |

Added translation as of https://github.com/fishpondstudio/CivIdle/pull/24/commits/9b893958592be33263f3b48e3fdbacd335443ce3

**Comment by insraq** [comment]
I actually remove the upgrade text in [0.1.124](https://github.com/fishpondstudio/CivIdle/commit/512790d50162fb3c5dae79690bb0a103893d7b3b) to clean up the visual clutter. But I guess it causes more confusion?

**Comment by jimmyh77** [comment]
> I actually remove the upgrade text in [0.1.124](https://github.com/fishpondstudio/CivIdle/commit/512790d50162fb3c5dae79690bb0a103893d7b3b) to clean up the visual clutter. But I guess it causes more confusion?

I do miss the text that told you how many resources you needed to upgrade, now you need to mouse over the button.

**Pull request merged**